addOns/pscanrules/src/main/java/org/zaproxy/zap/extension/pscanrules/CookieUtils.java

```diff
@@ -187,4 +192,40 @@ private static void validateParameterNotNull(Object parameter, String name) {
         }
         return ignoreList;
     }
+
+    /**
+     * Return the {@code true} if the cookie's "expire" value was set in the past, {@code false}
+     * otherwise. (Including if no "expires" value is set at all.)
```

Comment:
I’ll move the bracketed part inside the period and drop the capital so that it’s more clearly associated with the false condition.

Reply:
`Returns {@code true}...`